linux_os/guide/system/software/updating/ensure_GPG_keys_are_configured/rule.yml

```diff
@@ -1,57 +1,53 @@
+{{% if product in ["sle12", "sle15"] %}}
```

Comment:
This rule is restricted to `sle` products in `prodtype`. Also, I am not sure if other products are inclined to adopt this rule since it is a intentional `manual` requirement in CIS.

So I'm inclined to suggest the removal of any references to other products and keep it simpler and shorter just in case `sle`.

@dodys , @freddieRv and @teacup-on-rockingchair , what do you think?

Reply:
I agree @marcusburghardt. It does not make much sense to consider other products if the rule's `prodtype` is limited to SLE products only.

We don't have arguments against purely manual rules. If we ever need/want to include this one in an OL profile, we can make the pertinent adaptations.

Reply:
> I agree @marcusburghardt. It does not make much sense to consider other products if the rule's `prodtype` is limited to SLE products only.
> 
> We don't have arguments against purely manual rules. If we ever need/want to include this one in an OL profile, we can make the pertinent adaptations.

+1  on this from our side
If we consider using this rule we can provide the specific details as needed later on